types/jsftp/index.d.ts

```diff
@@ -0,0 +1,50 @@
+// Type definitions for jsftp 2.1
+// Project: https://github.com/sergi/jsftp
+// Definitions by: Konrad Księski <https://github.com/xyleen>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import { Socket } from 'net';
+import { EventEmitter } from 'events';
+
+export interface JsftpOpts {
+    host?: string;
+    port?: number;
+    user?: string;
+    pass?: string;
+    createSocket?: ({ port, host }: { port: number, host: string }, firstAction: () => {}) => Socket;
+    useList?: boolean;
+}
+
+export type ErrorCallback = (err: Error) => void;
+export type RawCallback = (err: Error, data: { code: number, text: string }) => void;
+export type ListCallback = (err: Error, dirContents: string) => void;
+export type GetCallback = (err: Error, socket: Socket) => void;
+export type LsCallback = (err: Error, res: [{ name: string }]) => void;
+
+export default class Ftp extends EventEmitter {
```

Comment:
It seems like the library not using `export default`, so this .d.ts file should not either.

Instead, you should follow the appropriate template from [the TypeScript handbook](http://www.typescriptlang.org/docs/handbook/declaration-files/templates.html).